src/services/deduplicationService.ts

```diff
@@ -0,0 +1,133 @@
+import { EnteFile } from 'types/file';
+import { getEndpoint } from 'utils/common/apiUtil';
+import { getToken } from 'utils/common/key';
+import { logError } from 'utils/sentry';
+import { getLocalFiles } from './fileService';
+import HTTPService from './HTTPService';
+
+const ENDPOINT = getEndpoint();
+
+interface DuplicatesResponse {
+    duplicates: Array<{
+        fileIDs: number[];
+        size: number;
+    }>;
+}
+
+interface DuplicateFiles {
+    files: EnteFile[];
+    size: number;
+}
+
+class DeduplicationService {
+    public async getDuplicateFiles() {
+        try {
+            const dupes = await this.fetchDuplicateFileIDs();
+            let ids: number[] = [];
+            for (const dupe of dupes) {
+                ids = ids.concat(dupe.fileIDs);
```

Comment:
we can use `map` for this na?

Reply:
also we don't use it anywhere ?

Reply:
yep my mistake, was using this earlier to test something and we don't need this now